plenum/bls/bls_key_register_pool_ledger.py

```diff
@@ -28,7 +29,9 @@ def _load_keys_for_root(self):
         for _, txn in self._ledger.getAllTxn():
             if get_type(txn) == NODE:
                 data = get_payload_data(txn)[DATA]
-                alias = data[ALIAS]
-                blskey = data.get(BLS_KEY)
-                keys[alias] = blskey
+                if not config.VALIDATE_SIGN_WITHOUT_BLS_KEY_PROOF and \
```

Comment:
Please write a warning that there is no BLS_KEY_PROOF

Reply:
Done